docs/developer-docs/smart-contracts/create.mdx

```diff
@@ -0,0 +1,63 @@
+---
+keywords: [beginner, tutorial, create, create canister]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+
+# Create
+
+<MarkdownChipRow labels={["Beginner", "Tutorial"]} />
+
+## Overview
+
+Canisters contain both code and state. Once you have written the code for a <GlossaryTooltip>canister</GlossaryTooltip>, the canister doesn't exist until it is registered with IC, either locally or the mainnet.
+
+A canister can be created without first compiling any code. When canisters are created with [`dfx canister create`](/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-canister#dfx-canister-create), they are empty and do not contain program code. The code must be compiled into Wasm and installed into the empty canister before it can be deployed.
+
+## Creating canisters 
+
+When a canister is created, either locally or on the mainnet, the following steps happen:
+
+- The canister ID is registered with the local replica or the mainnet for each canister in the project's `dfx.json` file.
+
+- The following canister components are created:
+      - List of controllers.
+      - Cycles balance.
+      - Reserved cycles balance.
+      - Canister status.
+      - Resource reservations.
+
+- A cycles wallet canister is created for the user principal that executed the command if one does not exist. This wallet canister's ID is returned in the command line output.
```

Comment:
does mdx hide code comments? we should add a TODO here to update when the cycles ledger is mature

Reply:
Yes, comments in `mdx` use:

{/* this is a comment */}

I've added one re: cycles ledger